openshift/ci-operator/generate-ci-config.sh

```diff
@@ -4,7 +4,7 @@ branch=${1-'knative-v0.6.0'}
 openshift=${2-'4.3'}
 promotion_disabled=${3-false}
 
-if [[ "$branch" -eq "knative-next" ]]; then
+if [[ "$branch" == "knative-next" ]]; then
```

Comment:
should the branch name be `release-next`, instead of `knative-next` ?

Reply:
@lberk   ^

Reply:
@matzew 
no, based on the way the branch/version variable is derived and passed in `update-ci` this is correct